docker/Dockerfile.ei

```diff
@@ -0,0 +1,10 @@
+ARG TFS_VERSION
+
+FROM 520713654638.dkr.ecr.us-west-2.amazonaws.com/sagemaker-tensorflow-serving:${TFS_VERSION}-cpu
```

Comment:
no do not use the prod repo! this breaks pr and release build flow. the image will be built from the previously released production image instead of the current image and we will ship a stale container!